src/schema/nmdc.yaml

```
@@ -2510,6 +2520,22 @@ enums:
       ZIC-cHILIC:
 
 slots:
+  eukaryotic_evaluation:
+    range: EukEval
+    description: Contains results from evaluating if a Metagenome-Assembled Genome is of eukaryotic lineage.
+  ncbi_lineage_tax_ids:
+    domain: EukEval
+    range: string
+    pattern: '^\d+(-\d+)*$'   
+    description: Dash delimited ordered list of NCBI taxonomy identifiers (TaxId)
```

```suggestion
    description: Dash-delimited ordered list of NCBI taxonomy identifiers (TaxId)
```

Reference (non-authoritative source, IMO):
> Generally, you need the hyphen only if the two or more words are functioning together as an adjective before the noun they’re describing. If the noun comes first, we usually leave the hyphen out.
- Source: https://www.grammarly.com/blog/hyphen/